shared/.dir-locals.el

```diff
@@ -1 +1,3 @@
-((clojure-mode . ((cider-preferred-build-tool . shadow-cljs))))
+((nil . ((cider-default-cljs-repl . shadow-select)
+         (cider-shadow-default-options . "node-repl")))
+ (clojure-mode . ((cider-preferred-build-tool . shadow-cljs))))
```

Comment:
you might want nil or clojurec-mode here. In the clojurec buffers i suspect you won't get the preferred-build-tool option

Reply:
`clojure-mode` is the parent mode of `clojurec-mode`, so it actually still works in that case. It's also the parent mode for `clojurescript-mode` so I guess these can all go under `clojure-mode`

Reply:
ok. i had forgotten if it inherited like that. i haven't used CIDER in a while so it wouldn't have impacted me.